### PR TITLE
Sample Data Ingestion use Create request

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on the Sample Data Ingestion use Create request.
Fix: #6433 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by pmbrull — APPROVED** [review]

**Comment by pmbrull** [comment]
Thanks @MilanBariya. Should we clean out the metadata_rest now? cc @ulixius9

**Comment by MilanBariya** [comment]
> Thanks @MilanBariya. Should we clean out the metadata_rest now? cc @ulixius9

I will clean it @pmbrull .

**Comment by ulixius9** [comment]
Let's clean amundsen and atlas as well to clean metadata_rest

File: ingestion/src/metadata/ingestion/source/database/sample_data.py [hunk 1]
```diff
@@ -409,83 +415,142 @@ def ingest_locations(self) -> Iterable[Location]:
...
-                    else LocationType.Iceberg
-                )
-            location_metadata = Location(
-                id=uuid.uuid4(),
+            table_request = CreateTableRequest(
                 name=table["name"],
-                path="s3://glue_bucket/dwh/schema/" + table["name"],
                 description=table["description"],
-                locationType=location_type,
+                columns=table["columns"],
```
Comment (ulixius9) [inline ingestion/src/metadata/ingestion/source/database/sample_data.py:462]: @MilanBariya here we are missing the information like `tableType`, `tableConstraints` which is available in [tables.json](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/examples/sample_data/datasets/tables.json)

File: ingestion/src/metadata/ingestion/source/database/sample_data.py [hunk 2]
```diff
@@ -409,83 +415,142 @@ def ingest_locations(self) -> Iterable[Location]:
...
-            table_and_db = OMetaDatabaseAndTable(
-                table=table_metadata, database=db, database_schema=schema
+            table_and_db = CreateTableRequest(
+                name=table["name"],
+                description=table["description"],
+                columns=table["columns"],
+                databaseSchema=EntityReference(
+                    id=database_schema_object.id, type="databaseSchema"
+                ),
+                tags=table["tags"],
```
Comment (ulixius9) [inline ingestion/src/metadata/ingestion/source/database/sample_data.py:550]: same info missing here as well

File: ingestion/src/metadata/ingestion/source/database/sample_data.py (hunk 1 above)
Comment (MilanBariya) [reply ingestion/src/metadata/ingestion/source/database/sample_data.py:462]: I will add that.

**Review by ulixius9 — APPROVED** [review]

**Pull request merged**